### PR TITLE
Feature/25577 new facets

### PR DESCRIPTION
**Pull request by cmoesgaard**
- [x] Have you updated the documentation?
- [x] Have you performed a smoke test of the application?
- [x] Have you written tests for your changes?
- [x] Have you updated the release notes?

### CODE REVIEW

File: docs/dev/facets.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+Facets
+======
+
+Facets serve as a grouping of different classes.
+Within the data models used in OS2MO, model fields corresponding to a facet
+name is expected to contain a reference to a class contained in that facet.
+E.g. the field ``responsibility`` in the ``manager`` data model, contains a
+reference to a class contained in the ``responsibility`` facet.
```
Comment (danchr) [inline docs/dev/facets.rst]: Mention that these are proper OIO objects? And that we disregard the actual classification?

File: docs/dev/facets.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+class from the ``manager_address_type`` facet. The ``address_type`` classes
+relevant in multiple contexts should be related to multiple facets, as
+opposed to duplicating the classes for each facet.
+
+The system currently works with the following facets:
+
++------------------------------+-----------------------------------------------------------+
+| Name                         | Description                                               |
++==============================+===========================================================+
+| ``org_unit_address_type``    | Address types applicable for organisational units         |
```
Comment (danchr) [inline docs/dev/facets.rst]:
```suggestion
| ``org_unit_address_type``    | Address types applicable to organisational units         |
```

etc.

File: docs/dev/facets.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
++------------------------------+-----------------------------------------------------------+
+| ``employee_address_type``    | Address types applicable for employees                    |
++------------------------------+-----------------------------------------------------------+
+| ``manager_address_type``     | Address types applicable for managers                     |
++------------------------------+-----------------------------------------------------------+
+| ``address_property``         | Properties for addresses, e.g. public or secret addresses |
```
Comment (danchr) [inline docs/dev/facets.rst]: `visibility`?

File: docs/index.rst [hunk 4]
```diff
@@ -9,6 +9,7 @@ Dette er den fuldstændige dokumentation til OS2MO 2.0 — MedarbejderOrganisati
 
    README
    cookbook
+   model
```
Comment (danchr) [inline docs/index.rst]: Leftover?

File: frontend/src/components/MoEntry/MoAddressEntry.vue [hunk 5]
```diff
@@ -85,7 +85,12 @@ export default {
     /**
      * Defines a preselectedType.
      */
-    preselectedType: String
+    preselectedType: String,
+
+    /**
+     * Defines a preselectedType.
+     */
+    facet: String
```
Comment (danchr) [inline frontend/src/components/MoEntry/MoAddressEntry.vue]: `required`?

File: frontend/src/components/MoEntry/MoEmployeeAddressEntry.vue [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+  components: {
+    MoAddressEntry
+  },
+
+  props: {
+    /**
+     * Create two-way data bindings with the component.
+     */
+    facet: {
```
Comment (andersjepsen) [inline frontend/src/components/MoEntry/MoEmployeeAddressEntry.vue]: you're not using this prop for anything

File: frontend/src/components/MoEntry/MoManagerAddressEntry.vue [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+  components: {
+    MoAddressEntry
+  },
+
+  props: {
+    /**
+     * Create two-way data bindings with the component.
+     */
+    facet: {
```
Comment (andersjepsen) [inline frontend/src/components/MoEntry/MoManagerAddressEntry.vue]: not using this prop for anything either

File: frontend/src/components/MoEntry/MoOrgUnitAddressEntry.vue [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+  components: {
+    MoAddressEntry
+  },
+
+  props: {
+    /**
+     * Create two-way data bindings with the component.
+     */
+    facet: {
```
Comment (andersjepsen) [inline frontend/src/components/MoEntry/MoOrgUnitAddressEntry.vue]: not using this prop for anything either

File: frontend/src/components/MoPicker/MoFacetPicker.vue [hunk 9]
```diff
@@ -1,7 +1,7 @@
 <template>
   <mo-select
     v-model="selected"
-    :label="facetData.user_key"
+    :label="$t('input_fields.'+facetData.user_key)"
```
Comment (andersjepsen) [inline frontend/src/components/MoPicker/MoFacetPicker.vue]: I prefter translations happening outside of the component. This way it's not dependent on having all its labels in a input_fields object

File: frontend/src/components/MoEntry/MoOrgUnitAddressEntry.vue [hunk 10]
```diff
@@ -0,0 +1,37 @@
+<template>
+  <mo-address-entry
+    facet="org_unit_address_type"
+    :value="value"
+    :validityHidden="validityHidden"
+    :required="required"
+    :label="label"
+    :preselectedType="preselectedType"
+  ></mo-address-entry>
```
Comment (andersjepsen) [inline frontend/src/components/MoEntry/MoOrgUnitAddressEntry.vue]: you can close it with />, no need for the extra </mo-address-entry>

**Pull request merged**